Router.js

```diff
@@ -14,23 +17,27 @@ const AppNavigator = StackNavigator({
     }
 })
 
-
+const splashScreenTimeout = 2000;
 export default class Router extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
             isLoggedIn: false,
-            disableRender:true
+            disableRender: true,
+            showSplashScreen: true
         }
     }
     async componentWillMount() {
```

Comment:
Can you move the await code to a separate function to retain the signature of componentWillMount lifecycle method ?